Docker/README.md

```diff
@@ -1361,77 +1403,75 @@ First have access to the application we developped for this.
-3gasrkzgpp0w  b            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_b    
-3sc3qexaixkl  redis        1/1       redis                                                 
-4c5i32juwnyh  myownsvc     1/1       lab7-2.labossi.hpintelco.org:5500/owncloud_web        
-5yl1168mm6h4  w2           1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_w2   
+1empjc9o6wwu  w            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_w
+1z53fru1vjr6  i            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_i
+3gasrkzgpp0w  b            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_b
+3sc3qexaixkl  redis        1/1       redis
+4c5i32juwnyh  myownsvc     1/1       lab7-2.labossi.hpintelco.org:5500/owncloud_web
+5yl1168mm6h4  w2           1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_w2
 6leldkqf1zth  ping         global    alpine                                                ping 8.8.8.8
-79jwqr43zyt2  web          1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_web  
-7hygz6g0lbyq  db           1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_db   
-9i4ogenk03ax  rabbit       1/1       rabbitmq:3-management                                 
+79jwqr43zyt2  web          1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_web
+7hygz6g0lbyq  db           1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_db
+9i4ogenk03ax  rabbit       1/1       rabbitmq:3-management
 ag12vg6ts417  tiny_curran  10/10     alpine                                                ping 8.8.8.8
-ajcrqc6nykn8  s            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_s    
-cn81a9a5j8yi  w1           1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_w1   
+ajcrqc6nykn8  s            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_s
+cn81a9a5j8yi  w1           1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_w1
 e6c6ypgcxdy2  p            1/1       lab7-2.labossi.hpintelco.org:5500/cloudnativeapp_p
 ```
 
 In order to use the application you'll now have to connect to http://c6.labossi.hpintelco.org/
 
-You should see a messsage in your browser saying:
+You should see a message in your browser saying:
 ```
 Please provide a user id !
 ```
 
-So now to use the application, you have to provide the id of the user who is playing to see his price.
+So now to use the application, you have to provide the id of the user who is playing to see his prize.
 Browse http://c6.labossi.hpintelco.org/index.html?id=1
 
 Check the availability of the application by restarting a docker daemon on a host running one of the containers the application is using.
-Check the micro-service behaviour by stopping the i micro-service, and then the p micro-service. Reload the Web page each time to see what happens.
+Check the micro-service behavior by stopping the 'I' micro-service, and then the 'P' micro-service. Reload the Web page each time to see what happens.
```

Comment:
Corrected.

I assumed you would prefer US spellings (e.g. behavior) rather than UK (e.g. behaviour).